Files/App.xaml.cs

```diff
@@ -300,6 +299,7 @@ protected override async void OnActivated(IActivatedEventArgs args)
                         var trimmedPath = eventArgs.Uri.OriginalString.Split('=')[1];
                         rootFrame.Navigate(typeof(InstanceTabsView), @trimmedPath, new SuppressNavigationTransitionInfo());
                     }
+                    ThemeHelper.Initialize();
```

Comment:
Don't need to duplicate command. Just move it to `InstanceTabsView` at the and of main constructor. And remove This command at all from App.xaml.cs.